src/EducationService.Business/Commands/Certificate/RemoveCertificateCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.EducationService.Business.Commands.Certificate.Interfaces;
+using LT.DigitalOffice.EducationService.Data.Interfaces;
+using LT.DigitalOffice.EducationService.Models.Db;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Threading.Tasks;
+using System.Net;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+
+namespace LT.DigitalOffice.EducationService.Business.Commands.Certificate
+{
+  public class RemoveCertificateCommand : IRemoveCertificateCommand
+  {
+    private readonly IAccessValidator _accessValidator;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly ICertificateRepository _certificateRepository;
+    private readonly IResponseCreater _responseCreator;
+
+    public RemoveCertificateCommand(
+      IAccessValidator accessValidator,
+      IHttpContextAccessor httpContextAccessor,
+      ICertificateRepository certificateRepository,
+      IResponseCreater responseCreator)
+    {
+      _accessValidator = accessValidator;
+      _httpContextAccessor = httpContextAccessor;
+      _certificateRepository = certificateRepository;
+      _responseCreator = responseCreator;
+    }
+
+    public async Task<OperationResultResponse<bool>> ExecuteAsync(Guid certificateId)
+    {
+      var senderId = _httpContextAccessor.HttpContext.GetUserId();
```

Comment:
please, do not use var

Reply:
I think better remove this variable and use _httpContextAccessor.HttpContext.GetUserId() in 41 line

Reply:
done